ksql-common/src/test/java/io/confluent/ksql/GenericRowTest.java

```diff
@@ -122,4 +128,50 @@ public void shouldPrintStructRowCorrectly() {
 
   }
 
-}
\ No newline at end of file
+  @Test
+  public void testEqualsReturningTrue() {
```

Comment:
Our general pattern for equals/hashcode tests is to use guava's `EqualsTester`. You can see `CreateStreamAsSelectTest` for an example